cfpb/design-manual
# Update for CF v5.0.5

Updates the Design Manual doc site to use the latest from Capital Framework.

## Changes

- Updates cf-notifications to 2.0.2

## Testing

- Run `./setup.sh`
- Run `npm start`
- Open `http://localhost:4000/page-components/form-fields.html#state-1` and see that the icons are no longer oversized (cf. [the live page](https://cfpb.github.io/design-manual/page-components/form-fields.html#state-1))

## Screenshots

### Before

<img width="350" alt="screen shot 2018-05-18 at 09 42 27" src="https://user-images.githubusercontent.com/1044670/40238624-bf5e1b10-5a81-11e8-8b15-3c57d858add1.png">

### After

<img width="350" alt="screen shot 2018-05-18 at 09 41 42" src="https://user-images.githubusercontent.com/1044670/40238641-c7ae26ac-5a81-11e8-94f4-cdafa1204163.png">
